### PR TITLE
Improve error messages when installing from manifest+archive

### PR DESCRIPTION
**Pull request by corneliusweig**
Related #358

### CODE REVIEW

File: pkg/download/fetch.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type HTTPFetcher struct{}
 func (HTTPFetcher) Get(uri string) (io.ReadCloser, error) {
 	resp, err := http.Get(uri)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "failed to get the uri %q", uri)
```
Comment (ahmetb) [inline pkg/download/fetch.go]: hmm do you think we could use wrapping `error` of FileFetcher.Get , like:

    failed to open specified archive file for reading: ..

**Comment by ahmetb** [comment]
/lgtm
/approve
/hold 
have only 1 comment.

File: pkg/download/fetch.go (hunk 1 above)
Comment (corneliusweig) [reply pkg/download/fetch.go]: Makes total sense, then this is treated uniformly in both FileFetchers.

**Pull request merged**